src/hooks/usePaymentDropIn.tsx

```diff
@@ -0,0 +1,31 @@
+import { lazy } from 'react';
```

Comment:
the `src/hooks` directory is meant to hold only context-agnostic, reusable hooks. this one is very context-specific, as it is used in only one component. please move it next to the component

The general rule is that a file should be placed in a location that is the "smallest" common parent location for all the other files that use it